crypto.go

```diff
@@ -1,21 +1,31 @@
 package libp2ptls
 
 import (
-	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
 	"crypto/rand"
 	"crypto/tls"
 	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
 	"errors"
 	"fmt"
 	"math/big"
 	"time"
 
+	crypto "github.com/libp2p/go-libp2p-crypto"
 	ic "github.com/libp2p/go-libp2p-crypto"
-	pb "github.com/libp2p/go-libp2p-crypto/pb"
 	peer "github.com/libp2p/go-libp2p-peer"
 )
 
-const certValidityPeriod = 180 * 24 * time.Hour
+const certValidityPeriod = 100 * 365 * 24 * time.Hour // ~100 years
+
+var extensionID = getPrefixedExtensionID([]int{1, 1})
```

Comment:
Should we export the extension ID?

Reply:
I don't see why we would need to do that at the moment, so I'd prefer to keep the API as small as possible.
By the way, what do you think about https://github.com/libp2p/specs/pull/151/files#diff-29603a9a6fd077382b3e7ccb9a175726R39?

Reply:
See response in https://github.com/libp2p/specs/pull/151/files#r264249948.